tests/cases/fourslash/renameModuleExportsProperties3.ts

```diff
@@ -0,0 +1,10 @@
+﻿/// <reference path='fourslash.ts' />
+
+// @allowJs: true
+// @Filename: a.js
+////[|class [|{| "contextRangeIndex": 0 |}A|] {}|]
+////module.exports = { [|A|] }
+
+const [r0Def, r0, r1] = test.ranges();
+verify.renameLocations(r0, { ranges: [r0, { range: r1, prefixText: "A: " }], providePrefixAndSuffixTextForRename: true });
+verify.renameLocations(r1, { ranges: [r0, { range: r1, prefixText: "A: " }], providePrefixAndSuffixTextForRename: true });

```

Comment:
Would you mind switching these tests from `verify.renameLocations` to `verify.baselineRename`? I find the former so difficult to parse.

Reply:
I used `renameLocations`, to test `prefixText`., I'm not sure that `baselineRename` allows inserting `prefixText` to baseline file., for instance, the baseline for the `renameModuleExportsProperties3.ts` looks like so,

```ts
/*====== /tests/cases/fourslash/a.js ======*/

class [|RENAME|] {}
module.exports = { RENAME }
```